### ISSUE TITLE
Import of depth profile data released under open licenses

### ISSUE THREAD

**Issue by Skippern** [OpenSeaMap/depth_webfrontend#125]
Is there a possibility to import depth data released under open licenses? It seems that the Norwegian [kartverket](http://www.kartverket.no/) either have or might soon release their depth data. They have a lot of data released under the terms of `Digital Norge`, and loads og geospatial data is already or in the process of being released under Public Domain or similar terms.

I have presented this in an issue under a different project, ([see here](https://github.com/Skippern/BSB_scripts/issues/7))

**Comment by cleanerx** [comment]
Before we start importing third party data we need to setup our processing chain and get more robustness into it. Otherwise importing and debugging huge amounts of data is really cumbersome. We have data from the German Hydrographic institute we could use as well but first we need to be able to generate meaningful contents from it.

**Comment by Skippern** [comment]
Ok.

Is it of interest to enter these datas into the pipe preparing for import, or should we keep it on hold until the processing chain is debugged and considered ready for release?

**Comment by cleanerx** [comment]
Well we need to analyze what kind of data their are providing. It seems to me that all the data is processed. Currently we focus on raw measured data rather than generated contour lines. I think we need a concept for this kind of integration that is yet to be designed. So unless it is raw data I think holding it off is the best way to handle it.

**Comment by Nautic** [comment]
We need a concept for to:
a) qualify raw data
b) qualify processed data
c) mark sources of data
d) mark licenses of data
e) store different quality levels, sources and licenses
f) store/transform different DB-formats

Until we have solutions, we should store such data seperate from our actual process,
added by a full description of source, format, quality, license, etc.

**Comment by HalluFIN** [comment]
There would be also open data available from Finland, released by Finish Trasportation Agency.
http://portal.liikennevirasto.fi/sivu/www/f/aineistopalvelut/avoin_data/Merikartoitus_lisenssi/License_hydrographic#.VZ4WgnLyVac
http://portal.liikennevirasto.fi/sivu/www/e/transport_network/nautical_charts/copyright

And I created a temporarily download link to the Agency, valid for 10 days counting from today.
https://extranet.liikennevirasto.fi/livi/inspire-downloads/c58e79bf-6054-4825-9429-46f2d07c1250.zip

**Comment by cleanerx** [comment]
Oh this license tricky. In strict terms I would be the licensee and I'm using the data to show them on the web page. Since I'm therefor redistributing it, it may be used for navigational purposes such as trip planning and positioning and this is exactly what the license forbids.

Is there any reason why this has been excluded ?
I think the section about waving off the liability would be ok but the other right is too restricted to make use of the data.

**Comment by HalluFIN** [comment]
I wrote to the ministerium. I'll wait the answer and get back to this issue.
